src/Core/Silk.NET.SilkTouch/NativeContextOverrideGeneration.cs

```diff
@@ -61,6 +63,41 @@ Compilation comp
                         var matchId = (int) x2.ConstructorArguments[0].Value!;
                         return matchId != attId;
                     })).ToArray(), comp.SyntaxTrees?.FirstOrDefault()?.IsNet5OrGreater() ?? false)));
+                arrLast = IfStatement
+                (
+                    InvocationExpression
+                    (
+                        MemberAccessExpression
+                        (
+                            SyntaxKind.SimpleMemberAccessExpression,
+                            IdentifierName("System.Linq.Enumerable"),
+                            IdentifierName("Contains")
+                        ),
+                        ArgumentList
+                        (
+                            SingletonSeparatedList
+                            (
+                                Argument
+                                (
+                                    IdentifierName("n")
+                                )
+                            ).Add
```

Comment:
It's been a minute since I touched this, but I believe using SingletonSeparatedList + Add isn't exactly best practice. I think you should just use SeparatedList?